### PR TITLE
kombu 5.2.2

### PR DESCRIPTION
**Pull request by skupr-anaconda**
**Current package info**

| Channel | Downloads | Version | Platforms | License |
| --- | --- | --- | --- | --- |
| defaults | [Conda Downloads](https://anaconda.org/main/kombu) | [Conda Version](https://anaconda.org/main/kombu) | [Conda Platforms](https://anaconda.org/main/kombu) | Conda License |

Changelog: https://github.com/celery/kombu/releases
License: https://github.com/celery/kombu/blob/v5.2.2/LICENSE
Requirements: https://github.com/celery/kombu/blob/v5.2.2/setup.py

Actions: 
1. Reset the build number
2. Apply a patch to fix `requirements/test.txt` file
4. Clean up the recipe
5. Add `--no-build-isolation` to `script`
6. Update runtime pinnings
7. Add license_family
8. Add description
9. Update doc_url

Notes:
- We update this package because of its incompatibility with **sqlalchemy 2.x.x**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,32 +1,34 @@
...
   host:
     - python
     - pip
+    - setuptools
+    - wheel
   run:
     - python
-    - importlib_metadata >=0.18  # [py<=37]
-    - amqp >=5.0.0,<6.0.0
+    - importlib_metadata >=0.18  # [py<38]
```
Comment (M-Waszkiewicz-Anaconda) [inline recipe/meta.yaml:31]: this is probably not needed

**Review by M-Waszkiewicz-Anaconda — APPROVED** [review]
LGTM

**Review by varlackc — COMMENTED** [review]
There is a list of related PRs that we might want to look at

  `kombu 5.2.0`
  https://github.com/AnacondaRecipes/kombu-feedstock/pull/1

  `kombu 5.2.1`
  https://github.com/AnacondaRecipes/kombu-feedstock/pull/2

  `kombu 5.2.2`
  https://github.com/AnacondaRecipes/kombu-feedstock/pull/3

  `kombu 5.2.3`
  https://github.com/AnacondaRecipes/kombu-feedstock/pull/4

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,32 +1,34 @@
...
-  sha256: f4965fba0a4718d47d470beeb5d6446e3357a62402b16c510b6a2f251e05ac3c
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/kombu-{{ version }}.tar.gz
+  sha256: 0f5d0763fb916808f617b886697b2be28e6bc35026f08e679697fc814b48a608
+  patches:
+    - fix_test_txt.patch
 
 build:
-  number: 1
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  number: 0
```
Comment (akabanovs) [inline recipe/meta.yaml:16]: I prefer to explicitly specify python requirements, even if our supported python versions already satisfy it. Here it is py>=37 and skip for older ones. But we never came to an agreement regarding this practice. Hence, up to you.

Comment (skupr-anaconda) [reply recipe/meta.yaml:16]: Fixed

File: recipe/meta.yaml (hunk 1 above)
Comment (skupr-anaconda) [reply recipe/meta.yaml:31]: It's only for reproducibility

File: recipe/meta.yaml [hunk 3]
```diff
@@ -35,14 +37,23 @@ test:
     - kombu.transport
     - kombu.transport.virtual
     - kombu.utils
+  requires:
+    - pip
+  commands:
```
Comment (akabanovs) [inline recipe/meta.yaml:53]: There are some unit tests present in the upstream. Do we want to add them?

Comment (skupr-anaconda) [reply recipe/meta.yaml:53]: I've added a test suite.

File: recipe/meta.yaml [hunk 4]
```diff
@@ -1,32 +1,35 @@
...
+  number: 0
+  skip: true  # [py<37]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   host:
     - python
     - pip
+    - setuptools
+    - wheel
```
Comment (akabanovs) [inline recipe/meta.yaml:28]: Are we missing patch as a requirement here? I mean in build section.

Comment (skupr-anaconda) [reply recipe/meta.yaml:28]: Thanks. Fixed it

**Review by akabanovs — APPROVED** [review]

**Pull request merged**